gtfs-realtime/spec/en/reference.md

```diff
@@ -1,14 +1,29 @@
 A GTFS Realtime feed lets transit agencies provide consumers with realtime information about disruptions to their service (stations closed, lines not operating, important delays, etc.) location of their vehicles, and expected arrival times.
 
-Version 1.0 of the feed specification is discussed and documented on this site.
+Version 2.0 of the feed specification is discussed and documented on this site.
 
 ### Term Definitions
 
-*   **required**: Exactly one
-*   **repeated**: Zero or more
```

Comment:
@RachM I had concerns about removing `repeated` too, but wasn't sure how to represent this in the table.  Currently I added text to the "Description" field, but I do like the ability to scan the table and easily see which fields are 0+ elements.

Questions for this:
* Should we add a new column for one/many relationships?  If so, what should this be called?  
* Do we leave this as the "Cardinality" column and change the name of the column for the new semantic fields to "Requirement", or something similar?

Also, I've never liked the term `repeated` for this concept because to my knowledge it's very protocol buffer (implementation)-specific.  I'd prefer to find another term - suggestions?